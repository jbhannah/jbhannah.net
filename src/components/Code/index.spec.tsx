import { render, screen } from "@testing-library/react"
import * as React from "react"
import { Code } from "."

const TestCode = (props = {}) => render(<Code data-testid="code" {...props} />)

describe("Code", () => {
  beforeEach(() => TestCode())

  it("renders correctly", () => {
    expect(screen.getByTestId("code")).toMatchSnapshot()
  })
})
