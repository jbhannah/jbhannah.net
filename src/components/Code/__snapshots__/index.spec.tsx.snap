// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Code renders correctly 1`] = `
<code
  class="css-em50vd"
  data-testid="code"
/>
`;
